Userland/Services/WindowServer/Menu.cpp

```diff
@@ -692,12 +692,21 @@ void Menu::set_visible(bool visible)
         m_client->async_menu_visibility_did_change(m_menu_id, visible);
 }
 
+void Menu::update_alt_shortcuts_for_items() {
```

Comment:
CI is complaining about this `{`. Please put it on its own line.

(If you install the [commit hook](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#commit-hooks) then git will tell you about formatting issues (and fix them) when you try to commit, instead of having to wait for CI to notice.)